api/numaresourcesoperator/v1/numaresourcesscheduler_normalize_test.go

```diff
@@ -61,6 +61,7 @@ func TestNUMAResourcesSchedulerSpecNormalize(t *testing.T) {
 				ScoringStrategy: &ScoringStrategyParams{
 					Type: scoringStrategyType,
 				},
+				Replicas: newInt32(3),
```

Comment:
nit: `ptr.To(int32(3))`
`newInt` does not imply it returns a pointer.

Reply:
yes, we should move (all the codebase ) to `ptr`: https://github.com/openshift-kni/numaresources-operator/issues/715

(the issue itself need to be updated)

I have a followup cleanup PR in the workings, I'll add this work to that PR